workshop/gamemodes/cinema_modded/gamemode/modules/theater/sh_commands.lua

```diff
@@ -113,6 +113,7 @@ else
 	CreateConVar( "cinema_allow_reset", 0, fcvar, "Reset the theater after all players have left." )
 	CreateConVar( "cinema_allow_voice", 0, fcvar, "Allow theater viewers to talk amongst themselves." )
 	CreateConVar( "cinema_allow_3dvoice", 1, fcvar, "Use 3D voice chat." )
+	CreateConVar( "cinema_force_extension_bypass", 0, fcvar, "Bypass file extention check when using direct file streaming" )
```

Comment:
Typo: "extention"

Also, you will probably have to add FCVAR_REPLICATED (like how cinema_queue_mode is set up at the top of the file) since clients also use SERVICE:Match (used in things like enabling/disabling the "request video" button).

Otherwise looks good :)

Reply:
Wouldn't it be a better idea to move `CreateConVar` to the top then?

Reply:
Yes